authentication-service/src/main/kotlin/com/saveourtool/save/authservice/config/WebSecurityConfig.kt

```diff
@@ -109,6 +109,7 @@ class WebSecurityConfig(
             "/api/$v1/contests/*/public-test",
             "/api/$v1/contests/*/scores",
             "/api/$v1/contests/*/*/best",
+            "/demo/api/*/run"
```

Comment:
In `gateway` we require authorization for whole `demo/api` but allow to `run`? Or what?

Reply:
Yes, I guess there could be some endpoints that might require authorization in the future
But `run` should be public

Reply:
But how we can reach `run` manually from frontend without auth, if all parent endpoints are require authorization?

Reply:
Did not get your point

If I got it right, any endpoint that matches the regex will be allowed for using with no authorization
There might be some special endpoints and so on, but simple demo run should not require authorization